test/modern_bpf/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 message(STATUS "Modern BPF tests build enabled")
 
+include_directories(../userspace/common)
```

Comment:
Could we use `${CMAKE_SOURCE_DIR}` to anchor this include a bit more?
```suggestion
include_directories(${CMAKE_SOURCE_DIR}/userspace/common)
```

Reply:
Question is, would that work if we build Falco with an external `FALCOSECURITY_LIBS_SOURCE_DIR`?

Reply:
Probably not 😅

Reply:
Was this concern addressed? :)

Reply:
I think so, we can keep the relative path for now.